crates/cdk/src/mint_url.rs

```diff
@@ -42,11 +63,6 @@ impl MintUrl {
         let url: Url = self.try_into()?;
         Ok(url.join(path)?)
     }
-
-    /// Remove trailing slashes from url
-    pub fn trim_trailing_slashes(&self) -> Self {
-        Self(self.to_string().trim_end_matches('/').to_string())
-    }
```

Comment:
Old unused code?

Reply:
Yeah we don't need this since we strip the `/` when creating the `MintUrl`